packages/sui-mono/bin/sui-mono-commit-all.js

```diff
@@ -37,16 +38,11 @@ const {message, type} = program
  * @param  {String}  path Folder to check
  * @return {Promise<Boolean>}
  */
-const hasChangedFiles = path => {
-  return new Promise((resolve, reject) => {
-    exec(
-      `git add . && git status ${path} --porcelain`,
-      {cwd: process.cwd()},
-      (err, {stdout}) => {
-        err ? reject(err) : resolve(stdout.trim() !== '')
-      }
-    )
-  })
+const hasChangedFiles = async path => {
+  const {stdout = ''} = await exec(
+    `git add . && git status ${path} --porcelain`
+  ).catch(() => '')
```

Comment:
Maybe some console.error would be nice when debugging errors..

Reply:
👍  I've added a way to log the error but still return an empty string.